regimes/sg/README.md

```diff
@@ -0,0 +1,75 @@
+
+Find example SG GOBL files in the [`examples`](../../examples/sg) (uncalculated documents) and [`examples/out`](../../examples/sg/out) (calculated envelopes) subdirectories.
+
+---
+
+## Overview of GST
+
+Singapore offers a simple GST model with a standard rate along with a few exceptions. It also offers a few methods for invoicing which will be described further down. Singapore also uses a wide variaty of TIN alternatives as their GST registration number. GST is handled by the Inland Revenue Authority of Singapore ([IRAS](https://www.iras.gov.sg/taxes/goods-services-tax-(gst)))
+
+---
+
+## Rates
+
+1. Standard rate of **9%**. (Since 01/01/2024)
+2. Zero-rate which applies to international services and export of goods.
+3. Exempt Supplies which include financial services, sale and lease of residential properties, digital payment tokens, and the import of investment precious metals.
+
+*Other tax rates such as 50% discount on selling price for second hand goods are not covered yet*
+
+## Invoicing methods
+
+There are three main methods for invoicing which will be described below. Other methods like credit notes and reverse billing have to follow the structure of a normal Tax Invoice.
+
+### Tax Invoice
+
+This invoice, reference in GOBL by the use of the tax tag "standard" represents a basic Invoice. This Invoice has to meet certain requirements:
+
+1. The words “tax invoice” in a prominent place.
+2. An identifying number (e.g. invoice number).
+3. Date of issue of the invoice.
+4. Supplier business name, address and GST registration number.
+5. Customer’s name and address.
+6. A description sufficient to identify the goods or services supplied and the type of supply.
+7. For each description of goods or services supplied, the quantity of goods or the extent of services, and the amount payable, excluding GST.
+8. Any cash discount offered.
+9. The total amount payable (excluding GST), the GST rate and the total amount of GST chargeable.
+10. The total amount payable (including the total amount of GSTchargeable).
+11. A breakdown of exempt, zero-rated or other supplies, stating separatelythe gross total amount payable in respect of each type of supply.
+
+### Simplified Tax Invoice
+
+This invoice is referenced by the tax tag "simplified". This invoice can only be used when the total amount (inclusive of GST) is less than $1000. This invoice has less requirements:
+
+1. Suplier name, address and GST registration number;
+2. An identifying number, e.g. invoice number.
+3. The date of issue of the invoice.
+4. Description of the goods or services supplied.
+5. The total amount payable including tax.
+6. The word “Price Payable includes GST”.
+
+### Reciept
+
+This type of invoice can be issued to a non-GST registered costumer. This invoice requires the following:
+
+1. Suplier name, address and GST registration number;
+2. The date of issue of the invoice.
+3. The total amount payable including tax.
+4. The word “Price Payable includes GST”.
```

Comment:
Based on this description, it feels like the receipt is probably not needed in GOBL, the "simplified" version will cover most cases.